src/pages/UserProfile/UserProfile.module.scss

```diff
@@ -0,0 +1,64 @@
+.userProfilePage {
+  margin: 0 auto;
+}
+
+.userBanner {
+  height: 12rem;
+  margin-bottom: 7rem;
+  background-color: $light-grey;
+
+  @media (min-width: $screen-lg-min) {
+    margin-bottom: 3rem;
+  }
+}
+
+.imageWrapper {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  margin-top: -12rem;
```

Comment:
There is always better way than negative margins. Once you go there, you never go back